templates/template.md.jinja

```diff
@@ -93,6 +93,112 @@ _Properties from [{{parent.label}}](#{{parent.shortname|urlencode}}):_ {% for pr
     height: auto;
     background-color: #fff;
 }
-.zoom { transition: transform ease-in-out 0.2s; }
-.zoom:hover { transform: scale(2.0); }
-</style>
\ No newline at end of file
+.zoom { position: relative; transition: transform ease-in-out 0.2s; z-index: 10; border: 2px solid #E6E6E6;}
+/* .zoom:hover { transform: scale(2.0); cursor: grab; } */
+.zoom:hover { transform: scale(2.0); cursor: grab; }
+.zoom:active { cursor: grabbing; }
+</style>
+<script>
```

Comment:
Van waar komt deze javascript en wat doet ze?

Reply:
Sommige diagrammen zijn te groot om op het scherm te tonen in de zoom versie. Deze maakt het mogelijk om de viewbox van de svg te verslepen. Gebaseerd [hierop](https://css-tricks.com/creating-a-panning-effect-for-svg/) maar nog wat aangepast.